extensions/typescript/src/features/documentHighlightProvider.ts

```diff
@@ -34,7 +34,18 @@ export default class TypeScriptDocumentHighlightProvider implements DocumentHigh
 		}
 		return this.client.execute('occurrences', args, token).then((response): DocumentHighlight[] => {
 			let data = response.body;
-			if (data) {
+			if (data && data.length) {
```

Comment:
We may also want to scope to using the TypeScript version just to be safe. That way if something does go terribly wrong, users could point to their own ts install as a workaround.